lib/generators.nix

```diff
@@ -203,19 +203,24 @@ rec {
     /* If this option is true, attrsets like { __pretty = fn; val = …; }
        will use fn to convert val to a pretty printed representation.
        (This means fn is type Val -> String.) */
-    allowPrettyValues ? false
-  }@args: let go = v: with builtins;
+    allowPrettyValues ? false,
+    /* If this option is true, the output is indented with newlines for attribute sets and lists */
+    multiline ? false
```

Comment:
should we flip to `true` by default? Makes more sense to me. Since it’s intended to be used in debug output, I don’t see how we could break user expectation if we split to multiline by default.